container/lxd/initialisation.go

```diff
@@ -49,6 +76,147 @@ func getPackagingConfigurer(series string) (config.PackagingConfigurer, error) {
 	return config.NewPackagingConfigurer(series)
 }
 
+func configureZFS() {
+	/* create a 100 GB pool by default (sparse, so it won't actually fill
+	 * that immediately)
+	*/
+	output, err := exec.Command(
+		"lxd",
+		"init",
+		"--auto",
+		"--storage-backend", "zfs",
+		"--storage-pool", "lxd",
+		"--storage-create-loop", "100",
```

Comment:
Doesn't this wipe the other settings like "core.https_address" ? This is probably fine for Juju interacting with LXD in a MaaS/EC2 provider, but for LXD (local) provider, we need to have LXD listen so that the Juju agent running in the container can talk to the host.
I guess maybe we do this early enough that we set https_address later?